python/lsst/ts/externalscripts/auxtel/latiss_take_twilight_flats.py

```diff
@@ -0,0 +1,151 @@
+        """Handle creating the camera object and waiting remote to start."""
+        if self.latiss is None:
+            self.log.debug("Creating Camera.")
+            self.latiss = LATISS(
+                self.domain, intended_usage=LATISSUsages.TakeImage, log=self.log
+            )
+            await self.latiss.start_task
+        else:
+            self.log.debug("Camera already defined, skipping.")
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_externalscripts/take_twilight_flats_latiss.yaml
+            title: TakeTwilightFlatsLatiss v1
+            description: Configuration for TakeTwilightFlatsLatiss.
+            type: object
+            properties:
+              filter:
+                description: Filter name or ID.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+                default: "SDSSr_65mm"
+              grating:
+                description: Grating name; if omitted the grating is not changed.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+                default: null
+              linear_stage:
+                description: Linear stage position; if omitted the linear stage is not moved.
+                anyOf:
+                  - type: number
+                  - type: "null"
+                default: null
+            additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema_yaml)
+
+        base_schema_dict = super(TakeTwilightFlatsLatiss, cls).get_schema()
+
+        for prop in base_schema_dict["properties"]:
+            schema_dict["properties"][prop] = base_schema_dict["properties"][prop]
+
+        return schema_dict
+
+    async def get_sky_counts(self) -> float:
+        """Abstract method to get the median sky counts from the last image.
+
+        Returns
+        -------
+        float
+            Sky counts in electrons.
+        """
+        # query consDB with util once such a function exists
+        sky_counts = 0
+        return sky_counts
+
+    def get_instrument_name(self) -> str:
+        """Get instrument name.
+
+        Returns
+        -------
+        instrument_name: `string`
+        """
+        return self.instrument_name
+
+    def get_instrument_configuration(self) -> dict:
+        return dict(
+            filter=self.config.filter,
+            grating=self.config.grating,
+            linear_stage=self.config.linear_stage,
+        )
+
+    def get_instrument_filter(self) -> str:
+        """Get instrument filter configuration.
+
+        Returns
+        -------
+        instrument_filter: `string`
+        """
+        filter = self.config.filter
+        grating = self.config.grating
```

Comment:
You can skip creating this local variable here and just use `self.config.grating` directly.